src/app/functions/server/onerep.ts

```diff
@@ -170,72 +189,34 @@ export async function createProfile(
 export async function getProfile(
   profileId: number,
 ): Promise<ShowProfileResponse> {
-  const response: Response = await onerepFetch(`/profiles/${profileId}/`, {
+  const response = await onerepFetch(`/profiles/${profileId}/`, {
     method: "GET",
   });
-  if (!response.ok) {
-    logger.error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-    throw new Error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-  }
 
-  const profile: ShowProfileResponse = await response.json();
-  return profile;
+  const profile = await response;
+
+  return profile as ShowProfileResponse;
 }
 
 export async function activateProfile(profileId: number): Promise<void> {
-  const response: Response = await onerepFetch(
-    `/profiles/${profileId}/activate`,
-    {
-      method: "PUT",
-    },
-  );
-  if (!response.ok) {
-    logger.error(
-      `Failed to activate OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-    throw new Error(
-      `Failed to activate OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-  }
+  await onerepFetch(`/profiles/${profileId}/activate`, {
+    method: "PUT",
+  });
+  logger.info("activated_onerep_profile", { onerepProfileId: profileId });
```

Comment:
Not sure if it matters or if we have an established style guide, but should these labels be suffixed or prefixed:
```
onerep_profile_xyz
```

or

```
xyz_onerep_profile
```

This isn't super-conclusive, but also, it's an admittedly kinda trash <kbd>grep</kbd>:
```sh
git grep -Pino "\blogger\.\w+\(\"[\w_-]+\"" src/ | grep "onerep_profile"

src/app/api/v1/admin/users/[primarySha1]/route.ts:187:logger.info("delete_onerep_profile"
src/app/api/v1/fxa-rp-events/route.ts:202:logger.info("deactivated_onerep_profile"
src/app/api/v1/fxa-rp-events/route.ts:294:logger.info("get_onerep_profile"
src/app/api/v1/fxa-rp-events/route.ts:312:logger.error("onerep_profile_not_found"
src/app/api/v1/fxa-rp-events/route.ts:359:logger.info("activated_onerep_profile"
src/app/api/v1/fxa-rp-events/route.ts:375:logger.error("onerep_profile_not_found"
src/app/api/v1/fxa-rp-events/route.ts:406:logger.info("deactivated_onerep_profile"
src/app/functions/server/onerep.ts:184:logger.info("onerep_profile_created"
src/app/functions/server/onerep.ts:205:logger.info("activated_onerep_profile"
src/app/functions/server/onerep.ts:212:logger.info("deactivated_onerep_profile"
src/app/functions/server/onerep.ts:219:logger.info("optout_onerep_profile"
```